src/system.c

```diff
@@ -1118,19 +1119,23 @@ void InitSystem (
     SyQuiet = 0;
     SyInitializing = 0;
 #ifdef SYS_IS_64_BIT
+    SyStorMin = 128*1024L;
     SyStorMax = 2048*1024L;          /* This is in kB! */
     SyAllocPool = 4096L*1024*1024;   /* Note this is in bytes! */
 #else
+    SyStorMin = 64*1024L;
     SyStorMax = 1024*1024L;          /* This is in kB! */
-#ifdef SYS_IS_CYGWIN32
+  #ifdef SYS_IS_CYGWIN32
     SyAllocPool = 0;                 /* works better on cygwin */
-#else
+  #else
     SyAllocPool = 1536L*1024*1024;   /* Note this is in bytes! */
-#endif
+  #endif
 #endif
     SyStorOverrun = 0;
     SyStorKill = 0;
-    SyStorMin = SY_STOR_MIN;         /* see system.h */
+#if SIZEOF_VOID_P == 8
```

Comment:
Is there supposed to be some code here?

Reply:
Ouch, that originally contained code to set `SyStorMin`, before I realized that it makes more sense to move that next to where we set `SyStorMax`.  Thanks for checking closely, a fixed version will be up in a few seconds.